test/e2e/helpers.js

```diff
@@ -643,6 +643,7 @@ const closeSRPReveal = async (driver) => {
   });
 };
 
+const DAPP_HOST_ADDRESS = '127.0.0.1:8080';
 const DAPP_URL = 'http://127.0.0.1:8080';
```

Comment:
Should we update this constant to use the above to keep them aligned?

Reply:
I do prefer to leave it as is since it's concise, rather than updating it to e.g. `DAPP_HOST_ADDRESS_WITH_SCHEMA`, as prepending the host address with the schema turns the value into a URL

Reply:
Sorry for the confusion, I meant should we update `DAPP_URL` to use `DAPP_HOST_ADDRESS`.

Reply:
ooh, sure! can do. updated b05d8569b5884ba20d196bc1cea5c74b79149341